admin/src/js/controllers/display-languages.js

```diff
@@ -21,11 +21,21 @@ angular.module('controllers').controller('DisplayLanguagesCtrl',
     'use strict';
     'ngInject';
 
-    const createLocaleModel = function(doc, totalTranslations) {
+    const createLocaleModel = function(doc, totalTranslations, enabledLocales) {
```

Comment:
TODO... settings structure - right now it's an array which means we can never extend it with any other info, eg: https://github.com/medic/cht-core/issues/8158